custom/icds_reports/data_pull/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class UnboundDataPull(Exception):
```

Comment:
I find it nice to have all exceptions end in `Exception` or `Error` just so it's clear that they are exceptions

Reply:
https://github.com/dimagi/commcare-hq/pull/26304/commits/7462b24f046acaed74e90a3f9a650df0b5781d44